stdlib/public/core/Algorithm.swift

```diff
@@ -12,18 +12,19 @@
 
 /// Returns the lesser of `x` and `y`.
 ///
-/// If `x == y`, returns `x`.
+/// - Important: If `x == y`, this function returns `x`.
```

Comment:
The `- Important:` field is used sparingly to highlight things that may cause grievous harm or other unexpected behavior. The documentation here (and below) doesn't meet that criteria, so please remove the callouts.